### PR TITLE
Reorganize the calculator demo

### PR DESCRIPTION
**Pull request by abarth**
This patch is a warmup to improving the visuals in the calculator demo.

Related to #4535

### CODE REVIEW

**Comment by abarth** [comment]
@HansMuller @Hixie

File: examples/flutter_gallery/lib/demo/calculator/home.dart [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+class KeyRow extends StatelessWidget {
+  KeyRow(this.keys);
+
+  final List<Widget> keys;
+
+  @override
+  Widget build(BuildContext context) {
+    return new Flexible(
+      child: new Row(
+        mainAxisAlignment: MainAxisAlignment.center, children: this.keys
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/calculator/home.dart]: Maybe a line break before the children: parameter.

File: examples/flutter_gallery/lib/gallery/app.dart [hunk 2]
```diff
@@ -12,7 +12,7 @@ import 'home.dart';
 final Map<String, WidgetBuilder> kRoutes = <String, WidgetBuilder>{
   PestoDemo.routeName: (BuildContext context) => new PestoDemo(),
   ShrineDemo.routeName: (BuildContext context) => new ShrineDemo(),
-  Calculator.routeName: (BuildContext context) => new Calculator(),
+  CalculatorDemo.routeName: (BuildContext context) => new CalculatorDemo(),
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/gallery/app.dart]: Thanks for making this consistent.

**Comment by HansMuller** [comment]
The calculator demo should be added to flutter_gallery/test_driver/transitions_perf_test.dart.  It looks like Pesto needs to be added as well.

I suppose it's time to make the list of demo button labels a computed value.

**Comment by HansMuller** [comment]
LGTM

File: examples/flutter_gallery/lib/demo/calculator/home.dart (hunk 1 above)
Comment (abarth) [inline examples/flutter_gallery/lib/demo/calculator/home.dart]: Yep.  :)

**Comment by abarth** [comment]
I've left the restructure of the gallery app for another patch.  :smile:

**Pull request merged**